### ISSUE TITLE
bug: `npm i` is not translated 🤷🏻‍♂️

### ISSUE THREAD

**Issue by ScottAgirs** [sapphiredev/documentation-plugins#109]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the bug

When abbreviating `npm install` to `npm i`:
```bash
  npm i keystone-6-oauth
```

the result is:
<img width="271" alt="Screen Shot 2022-10-05 at 9 43 59 AM" src="https://user-images.githubusercontent.com/11737572/193997383-9786a4d6-3e9a-4ff6-a457-8db8446e58f0.png">
<img width="316" alt="Screen Shot 2022-10-05 at 9 44 11 AM" src="https://user-images.githubusercontent.com/11737572/193997406-c242a56c-f326-4599-9b8f-fbeb90220d03.png">
<img width="319" alt="Screen Shot 2022-10-05 at 9 44 21 AM" src="https://user-images.githubusercontent.com/11737572/193997429-380c1b54-5015-4701-b8af-b258dbb378f6.png">

With `npm install ...`, however, all works as expected.

Just thought i'll leave this here, in case this is an oversight :)

### Steps To Reproduce

Use 

```bash
  npm i keystone-6-oauth
```

instead of 

```bash
  npm install keystone-6-oauth
```

### Expected behavior

`docusaurus.config.js`:
```js
{
// ...
  docs: {
    remarkPlugins: [npm2yarn2pnpm],
  },
// ...
```

`"@sapphire/docusaurus-plugin-npm2yarn2pnpm": "^1.1.3",`
`"@docusaurus/core": "2.1.0",`
`"@docusaurus/preset-classic": "2.1.0",`

**Comment by favna** [comment]
If someone wants to make a PR to fix this then that's be awesome.

**Comment by favna** [comment]
Released in v1.1.4
